ova/generate_ova.sh

```diff
@@ -19,9 +20,9 @@ scriptpath=$(
   pwd -P
 )
 
-function help() {
+help () {
 
-  OPTIONS:
+  echo "OPTIONS:"
```

Comment:
Can you change this line to these lines: https://github.com/wazuh/wazuh-packages/blob/master/debs/generate_debian_package.sh#L110-L113. This will keep consistency between the help messages in all the different scripts.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/223/commits/2629f18c2d161e7cfe70b71d39522548b5459f55